modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbStorageEngine.java

```diff
@@ -69,6 +70,11 @@ public class RocksDbStorageEngine implements StorageEngine {
             new NamedThreadFactory("rocksdb-storage-engine-pool", LOG)
     );
 
+    private final ScheduledExecutorService scheduledPool = Executors.newScheduledThreadPool(
```

Comment:
you can use `newSingleThreadScheduledExecutor` here

Reply:
Guess I can